wdl/methods/phasing/PhysicalAndStatisticalPhasing.wdl

```diff
@@ -47,6 +47,13 @@ workflow PhysicalAndStatisticalPhasing {
         locus = region
     }
 
+    call FilterPhasingInfo as FilterPhasing { input:
+        sv_vcf = SubsetVcfSV.subset_vcf,
+        sv_vcf_tbi = SubsetVcfSV.subset_tbi,
+        prefix = prefix + "filter_phasing"
```

Comment:
This does not respect the existing prefix convention. Since this will only be applied to the SV VCF, I think `prefix + ".unphased"` is fine.

Reply:
Note also that I neglected to change the prefix convention in ConvertLowerCase. I think `~{samplename}.uppercased` would be most succinct there.

Note also that some tasks enforce the prefix convention in their output lines (as does ConvertLowerCase), while others expose them and enforce the convention in the call to the task. Perhaps we should do this more consistently.